src/print_event_cb.c

```diff
@@ -296,11 +302,11 @@ fwrite_out_lf_fld_sep(FILE *f)
 void
 init_printing_events(void)
 {
-	static char str[] = "_----------------_----------------_";
+	static char str[] = "_------------------_------------------_";
+	char *c;
 
-	str[0]  = Args.separator;
-	str[17] = Args.separator;
-	str[34] = Args.separator;
+	while (c = strchr(str, '_'))
+		*c = Args.separator;
```

Comment:
Oh wait, this `str` might not count as something holding a string literal, gotta look into it!